File: gluecodium/src/test/resources/smoke/nesting/output/android/com/example/smoke/FreePoint.java
/*
 *

 */
package com.example.smoke;
import android.support.annotation.NonNull;
public final class FreePoint {
    public static final FreeEnum A_BAR = FreeEnum.BAR;
    public double x;
    public double y;
    public FreePoint(final double x, final double y) {
        this.x = x;
        this.y = y;
    }
    @NonNull
    public native FreePoint flip();
}

File: gluecodium/src/test/resources/smoke/nesting/output/android-kotlin/com/example/smoke/FreePoint.kt
/*

 *
 */

@file:JvmName("FreePoint")

package com.example.smoke


class FreePoint {
    @JvmField var x: Double
    @JvmField var y: Double



    constructor(x: Double, y: Double) {
        this.x = x
        this.y = y
    }



    external fun flip() : FreePoint


    companion object {
        val A_BAR: FreeEnum = FreeEnum.BAR
    }
}

